### DISCUSSION TITLE
App icon

### DISCUSSION THREAD

**Discussion by AYL-G** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#88] [Q&A]
Can I use the alternative icon in the task manager? Where it shows the startup apps and the running processes.

Task manager photos:
![image](https://user-images.githubusercontent.com/92548269/194695861-a759d079-fd75-417f-bfc0-d783fa1ec6c3.png)
![image](https://user-images.githubusercontent.com/92548269/194695879-c769f260-8e7e-4313-80af-9d7e18c14e5d.png)

Alternative icon:
![image](https://user-images.githubusercontent.com/92548269/194695905-740a4938-eb34-400b-814f-3dbe7eb09167.png)

**Comment by HerMajestyDrMona** [comment]
I read [this answer](https://stackoverflow.com/a/35135129/2517973) on stackoverflow, which claims that Windows is slow to re-cache icons.

The easiest workaround that worked for me:
- Copy `Windows11DragAndDropToTaskbarFix.exe` or `Windows11DragAndDropToTaskbarFix_Static.exe` file to clipboard.
- Go to Desktop, click with the Right Mouse Button -> More Options -> Paste Shortcut.
- Click with the Right Mouse Button on the newly created icon, then "Properties" from the list.
- Go to the "Shortcut" tab, and click on the "Change Icon..." button.  Then select the alternative icon and save changes.
- After that, start the program using the new shortcut.

![Alternative Icon](https://user-images.githubusercontent.com/24290461/194697355-5d93f864-ddbc-42de-b6d9-cf65142109f7.png)
